cypress/integration/ErrorHandling/MissingAction.spec.js

```diff
@@ -10,44 +10,66 @@ import * as editDialogModal from '../../support/components/EditDialogModal'
 import * as common from '../../support/Common'
 import * as actions from '../../support/Actions'
 import * as scorerModal from '../../support/components/ScorerModal'
+import * as helpers from '../../support/Helpers'
 
-describe('ErrorHandling', () => {
-  it('Missing Action', () => {
-    models.ImportModel('z-missingAction', 'z-whatsYourName.cl')
-    modelPage.NavigateToTrainDialogs()
-    cy.WaitForTrainingStatusCompleted()
-
-    modelPage.VerifyNoErrorIconOnPage()
+describe('Missing Action - ErrorHandling', () => {
+  afterEach(helpers.SkipRemainingTestsOfSuiteIfFailed)
+  
+  context('Setup', () => {
+    it('Should import a model and wait for training to complete', () => {
+      models.ImportModel('z-missingAction', 'z-whatsYourName.cl')
+      modelPage.NavigateToTrainDialogs()
+      cy.WaitForTrainingStatusCompleted()
+    })
 
+    it('Should verify there are no error icons on the page and create a new Train Dialog', () => {
+      modelPage.VerifyNoErrorIconOnPage()
     train.CreateNewTrainDialog()
```

Comment:
Minor, but it's slightly more appropriate to be grouped with the next test which modifies the dialog

Reply:
Done.